docs/en/sql-reference/statements/create/view.md

```diff
@@ -90,6 +92,42 @@ Views look the same as normal tables. For example, they are listed in the result
 
 To delete a view, use [DROP VIEW](../../../sql-reference/statements/drop.md#drop-view). Although `DROP TABLE` works for VIEWs as well.
 
+## SQL security {#sql_security}
+
+`DEFINER` and `SQL SECURITY` allow you to specify which ClickHouse user to use when executing the view's underlying query.
+`SQL SECURITY` has three legal values: `DEFINER`, `INVOKER`, or `NONE`. You can specify any existing user or `CURRENT_USER` in the `DEFINER` clause.
+
+The following table will explain which rights are required for which user in order to select from view. 
+Note that regardless of the SQL security option, in every case it is still required to have `GRANT SELECT ON <view>` in order to read from it.
+
+| SQL security option | View                                                            | Materialized View                                                                                                 |
+|---------------------|-----------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| `DEFINER alice`     | `alice` must have a `SELECT` grant for the view's source table. | `alice` must have a `SELECT` grant for the view's source table and an `INSERT` grant for the view's target table. |
+| `INVOKER`           | User must have a `SELECT` grant for the view's source table.    | `SQL SECURITY INVOKER` can't be specified for materialized views.                                                 |
+| `NONE`              | -                                                               | -                                                                                                                 |
+
+:::note
+`SQL SECURITY NONE` is a deprecated option. Any user with the rights to create views with `SQL SECURITY NONE` will be able to execute any arbitrary query.
+Thus, it is required to have `GRANT ALLOW SQL SECURITY NONE TO <user>` in order to create a view with this option.
+:::
+
+If `DEFINER`/`SQL SECURITY` weren't specified, the default values will be used:
+- `SQL SECURITY`: `DEFINER` ([configurable by settings](../../../operations/settings/settings.md#default_view_sql_security))
+- `DEFINER`: `CURRENT_USER` ([configurable by settings](../../../operations/settings/settings.md#default_view_definer))
+
+### Examples sql security
+```sql
+CREATE test_view
+DEFINER = alice SQL SECURITY DEFINER
+AS SELECT ...
+```
+
+```sql
+CREATE test_view
+SQL SECURITY INVOKER
+AS SELECT ...
+```
+
```

Comment:
Please add also documentation about new privilege to
`docs/en/sql-reference/statements/grant.md`

And also it would be nice to write documentation in Russian (because you can).